src/docs/reference/deployments.md

```diff
@@ -54,7 +58,57 @@ A Deployment will remain in the `Active` state unless it [crashes](/guides/deplo
 
 #### Removed
 
-When a new Deployment is triggered, older deploys in a `Crashed` or `Active` state are eventually removed - first having their status updated to `Removing` before they are finally `Removed`. Deployments may also be removed manually.
+When a new [Deployment](/overview/the-basics#deployments) is triggered, older deploys in a `Active`, `Completed`, or a `Crashed` state are eventually removed - first having their status updated to `Removing` before they are finally `Removed`. Deployments may also be [removed manually](/reference/deployments#remove).
+
+The time from when a new deployment becomes `Active` until the previous deployment is removed can be controlled by setting a [`RAILWAY_DEPLOYMENT_OVERLAP_SECONDS`](/reference/variables#user-provided-configuration-variables) [service variable](/overview/the-basics#service-variables).
+
+## Deployment Menu
+
+The deployment menu contains actions you can take on a deployment.
+
+**Note:** Some actions are only available on **Active** or **Completed** deployments.
+
+<Image
+  src="https://res.cloudinary.com/railway/image/upload/v1726503037/docs/redeploy_remove_deploy_jescm0.png"
+  alt="Deployment Menu"
+  width={1007}
+  height={690}
+  quality={80}
+/>
+
+#### View logs
+
+Opens the deployment up to the corresponding logs, during build the build logs will be shown, during deploy the deploy logs will be shown.
+
+#### Restart
+
+Restarts the currently running process in the deployment, this is often used to bring a service back online after a crash or if you application has locked up.
+
+#### Redeploy
```

Comment:
selfish, but can you add a note about how we only retain images for 2 weeks. i.e. you can only roll back to a deployment that is 2 weeks old or newer (ref [LOG-2359](https://linear.app/railway/issue/LOG-2359/update-docs-to-include-2-week-retention-period-for-container-images))